app/src/componentLibrary/sidebar/sidebar.jsx

```
@@ -74,21 +74,24 @@ export const Sidebar = ({
-                    message={message}
-                  />
-                ),
-            )}
+            {items.map(({ icon, link, onClick, message, component }) => {
+              const handleClick = () => {
+                onClick(getIsSidebarCollapsed());
+                onLeaveSidebar();
+              };
+
+              return component ? (
+                <div onClick={handleClick}>{component}</div>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)